packages/verifiable-credentials/src/credentials.ts

```diff
@@ -68,11 +68,16 @@ export default class Credentials {
 
 		const { data, schema } = createCredentialData
 
+		const {
+			vc,
+			issuer
+		} = await this.buildVerifiableCredential(createCredentialData)
```

Comment:
The current:
```ts
const didJwtVc = await this.createCredentialJWT(createCredentialData)

const {
	vc,
	issuer
} = await this.buildVerifiableCredential(createCredentialData)
```

Can be optimised to:
```ts
const {
	vc,
	issuer
} = await this.buildVerifiableCredential(createCredentialData)

const didJwtVc = await this.createVerifiableCredential(vc, issuer)
```

As `createCredentialJWT` execute `buildVerifiableCredential` internally, so `buildVerifiableCredential` is currently done twice.